mount/mount_linux.go

```diff
@@ -93,6 +107,22 @@ func Unmount(target string, flags int) error {
 }
 
 func unmount(target string, flags int) error {
+	inf, err := Lookup(target)
```

Comment:
If we want to avoid `Lookup` overhead, can we rely on `FUSE_SUPER_MAGIC`?

Reply:
```
func BenchmarkLookup(b *testing.B) {
        for i := 0; i < b.N; i++ {
                mount.Lookup("/var/lib/containerd")
        }
}

BenchmarkLookup-4            674           1688564 ns/op # 268 mountpoints 40pods
BenchmarkLookup-4           1087           1039448 ns/op # 168 mountpoints 20pods
BenchmarkLookup-4           3290            361808 ns/op # 68 mountpoints 1pod
```

In k8s env, there will be a lot of mount points. I am not a fan of calling `Lookup` for each umount. We can try the `FUSE_SUPER_MAGIC` or introduce high level service for mount management, like https://github.com/containerd/containerd/pull/3717

Reply:
updated PR to use FUSE_SUPER_MAGIC